Core/Src/main.c

```diff
@@ -124,6 +125,12 @@ int _write(int file, char* ptr, int len) {
   }
   return len;
 }
+
+/*Callback for UART*/
+void HAL_UART_TxCpltCallback(UART_HandleTypeDef *phuart) { //jake

```

Comment:
If you intend for "// jake" to be a permanent comment, use a multiline (/* */) as that is our standard

Reply:
Since you defined this function, it needs a java-doc style doxygen comment saying it's purpose. See other function defs (look in header files) for an example (or google it if the examples in the codebase aren't good enough).